secant/Util/InitialisationState.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  InitialisationState.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 30.03.2022.
+//
+
+import Foundation
+
+enum InitialisationState: Equatable {
+    case failure(ErrorWrapper)
+    case initialized
+    case keysMissingFilesPresent
```

Comment:
[non-blocking] just FYI
On ECC Reference Wallet we called this state "Unprepared" but it's ok to leave it as it is for now since the `prepare` step on the sdk may not be around for the next iteration :)

Reply:
Noted, similarly, seems like there is no action point at the moment. Let's wait on how the SDK evolves.